content/docs/data-api/demo.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Neon Data API tutorial
+subtitle: Set up our demo note-taking app to learn about postgrest-js queries
+enableTableOfContents: true
+---
+
+<FeatureBetaProps feature_name="Neon Data API" />
+
+In this tutorial, we'll walk through our note-taking app to show how you can use Neon's Data API and `postgrest-js` to write queries from your frontend code, with proper authentication and Row-Level Security (RLS) policies ensuring your data stays secure.
+
+## About the sample application
+
+This note-taking app is built with React and Vite. It uses Neon Auth for authentication, the Data API for direct database access, and Drizzle ORM for handling the schema.
+
+![Notes app UI](/docs/data-api/all_notes.png)
```

Comment:
Minor: I'm not entirely sure why you get generated note names here but not in the screencaps below.